controllers/disruption_controller.go

```diff
@@ -114,6 +114,29 @@ func (r *DisruptionReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		}
 	}()()
 
+	defer func() {
+		if err == nil {
+			return
+		}
+
+		unwrappedError, ok := err.(chaostypes.DisruptionError)
+		if ok {
+			errorContext := unwrappedError.Context()
+
+			if isModifiedError(unwrappedError) {
+				r.log.Infow(fmt.Sprintf("retryable %s", unwrappedError.Error()), errorContext)
```

Comment:
Aren't we suppose to have key values here?
```suggestion
				r.log.Infow("a retryable error occured in reconcile loop", "err", unwrappedError, "context", errorContext)
```

An alternative could be:
```suggestion
                                 wraps := make([]interface{}, 0, len(errorContext)+2)
                                 wraps = append(wraps, "err", unwrappedError)
                                 for key, value := range errorContext {
                                   wraps = append(wraps, key, value)
                                 }
				r.log.Infow("a retryable error occured in reconcile loop", wraps...)
```